### PR TITLE
Fix misleading allingment in job template editor

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/15190

### CODE REVIEW

**Comment by coolo** [comment]
hard to follow the change, but I trust you there

**Comment by Martchus** [comment]
Also if you only check the first commit?

File: assets/javascripts/job_templates.js [hunk 1]
```diff
@@ -106,7 +106,14 @@ function addTestRow() {
     $('<td class="prio">50</td>').appendTo(tr);
 
     var archnames = table.data('archs');
-    $.each(archnames, function(ai, arch) {
+    var archHeaders = table.find('thead th.arch');
+    var archColIndex = 0;
+    $.each(archnames, function(archIndex, arch) {
+        while (archColIndex < archHeaders.length
```
Comment (Martchus) [inline assets/javascripts/job_templates.js]: This while loop is the fix. The other changed are mostly refactoring because I thought it would be required in the first place.

**Comment by coolo** [comment]
it's still a lot of code

**Comment by okurz** [comment]
this is how it looks in production:
openqa_fixed_job_templates_alignment
